save/43786.0.txt

```diff
@@ -0,0 +1,73 @@
+@0
+
+
+
+#starts the loop
+@1
+START: MOV UP, ACC
+
+#swaps the "A" value
+A:SWP
+
+MOV RIGHT, ACC
+
+#if the value is equal to 0, loop restarts
+JEZ START
+
+#swaps the value
+SWP
+
+MOV ACC, RIGHT
+
+JMP A
+
+
+@2
+STRT: MOV UP ACC
+
+#if the "A" value is "y" move 0 left
```

Comment:
Seems to be a misunderstanding of what the code does here. You're checking if the value of `ACC` is zero, and if it is then jumping to the label `Y` (which does indeed move 0 to the left). `A:` here is a label, the target of a jump from elsewhere.